sdk/storage/azure-storage-blob/out/azure/storage/blob/_models.pyi

```diff
@@ -0,0 +1,231 @@
+    def __init__(self, command: Any, container: Optional[Any] = ..., prefix: Optional[Any] = ..., results_per_page: Optional[Any] = ..., continuation_token: Optional[Any] = ..., delimiter: Optional[Any] = ..., location_mode: Optional[Any] = ...) -> None: ...
+
+class BlobPrefix(ItemPaged, DictMixin):
+    name: Any = ...
+    prefix: Any = ...
+    results_per_page: Any = ...
+    container: Any = ...
+    delimiter: Any = ...
+    location_mode: Any = ...
+    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
+
+class BlobPrefixPaged(BlobPropertiesPaged):
+    name: Any = ...
+    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
+
+class LeaseProperties(DictMixin):
+    status: Any = ...
+    state: Any = ...
+    duration: Any = ...
+    def __init__(self, **kwargs: Any) -> None: ...
+
+class ContentSettings(DictMixin):
+    content_type: Any = ...
+    content_encoding: Any = ...
+    content_language: Any = ...
+    content_md5: Any = ...
+    content_disposition: Any = ...
+    cache_control: Any = ...
+    def __init__(self, content_type: Optional[Any] = ..., content_encoding: Optional[Any] = ..., content_language: Optional[Any] = ..., content_disposition: Optional[Any] = ..., cache_control: Optional[Any] = ..., content_md5: Optional[Any] = ..., **kwargs: Any) -> None: ...
+
+class CopyProperties(DictMixin):
+    id: Any = ...
+    source: Any = ...
+    status: Any = ...
+    progress: Any = ...
+    completion_time: Any = ...
+    status_description: Any = ...
+    incremental_copy: Any = ...
+    destination_snapshot: Any = ...
+    def __init__(self, **kwargs: Any) -> None: ...
+
+class BlobBlock(DictMixin):
+    id: Any = ...
+    state: Any = ...
+    size: Any = ...
+    def __init__(self, block_id: Any, state: Any = ...) -> None: ...
+
+class PageRange(DictMixin):
+    start: Any = ...
+    end: Any = ...
+    def __init__(self, start: Optional[Any] = ..., end: Optional[Any] = ...) -> None: ...
+
+class AccessPolicy(GenAccessPolicy):
+    start: Any = ...
+    expiry: Any = ...
+    permission: Any = ...
+    def __init__(self, permission: Optional[Any] = ..., expiry: Optional[Any] = ..., start: Optional[Any] = ...) -> None: ...
+
+class ContainerSasPermissions:
+    read: Any = ...
+    write: Any = ...
+    delete: Any = ...
+    list: Any = ...
+    def __init__(self, read: bool = ..., write: bool = ..., delete: bool = ..., list: bool = ...) -> None: ...
+    @classmethod
+    def from_string(cls, permission: Any): ...
+
+class BlobSasPermissions:
+    read: Any = ...
+    add: Any = ...
+    create: Any = ...
+    write: Any = ...
+    delete: Any = ...
+    def __init__(self, read: bool = ..., add: bool = ..., create: bool = ..., write: bool = ..., delete: bool = ...) -> None: ...
+    @classmethod
+    def from_string(cls, permission: Any): ...
+
+class ContainerCpkScopeInfo(ContainerCpkScopeInfoGen):
+    default_encryption_scope: Any = ...
+    deny_encryption_scope_override: Any = ...
```

Comment:
This property should be renamed to align with .NET/Java to `prevent_encryption_scope_override`